mason/mason.go

```diff
@@ -59,10 +59,83 @@ func writeFileJson(path string, content []byte) bool {
 	return true
 }
 
-func getProjects(projectId string, all bool, output string) {
+func exportProject(projectId string, output string) {
+	var outputSchema = ""
+	var outputContent = ""
+	var projectPath = ""
+	if output != "" {
+		if err := os.MkdirAll(output, os.ModePerm); err != nil {
+			fmt.Printf("\nError making directory %s\n", output)
+		}
+		projectPath = output + "/" + projectId
+		schemaPath := projectPath + "/schemas"
+		contentPath := projectPath + "/content"
+		if err := os.MkdirAll(schemaPath, os.ModePerm); err != nil {
+			fmt.Printf("\nError making directory %s\n", schemaPath)
+		}
+		if err := os.MkdirAll(contentPath, os.ModePerm); err != nil {
+			fmt.Printf("\nError making directory %s\n", contentPath)
+		}
+		outputSchema = schemaPath + "/schemas.json"
+		outputContent = contentPath + "/content.json"
+	}
+	getSchema("", false, outputSchema, projectId)
+	getContent("", false, outputContent, projectId)
+	if output != "" {
+		fmt.Printf("Wrote Project Schema and Content details to %s\n", projectPath)
+	}
+}
+
+func importProject(projectPath string, schemaOnly bool, contentOnly bool) {
+	if !schemaOnly && !contentOnly {
+		schemaOnly = true
+		contentOnly = true
+	}
```

Comment:
What is this doing? Are we saying we will accept any values in schemaOnly / contentOnly

Reply:
There was some issue parsing schemaOnly and contentOnly as default `true` in flagset.
That's why this, if no option is sent, both `schema` and `content` are imported